### ISSUE TITLE
Implement remaining `ewm` functions (relative to `pandas`)

### ISSUE THREAD

**Issue by wangkev** [pola-rs/polars#9223]
### Problem description

Would be great to implement remainder `ewm` functions that `pandas` has? I believe `sum`, `corr`, and `cov` are still missing.

Original `ewm` discussion: https://github.com/pola-rs/polars/issues/2148

**Comment by sahibble** [comment]
Agreed! There seem to already be rust implementations of `ewm_cov` and you can get `corr` for free when you compute `cov`!
